src/cargo/util/network/retry.rs

```diff
@@ -108,9 +108,11 @@ impl<'a> Retry<'a> {
                     .downcast_ref::<HttpNotSuccessful>()
                     .map(|http_err| http_err.display_short())
                     .unwrap_or_else(|| e.root_cause().to_string());
+                let left_retries = self.max_retries - self.retries;
                 let msg = format!(
-                    "spurious network error ({} tries remaining): {err_msg}",
-                    self.max_retries - self.retries,
+                    "spurious network error ({} {} remaining): {err_msg}",
+                    left_retries,
+                    if left_retries > 1 { "tries" } else { "try" }
```

Comment:
Can you switch this to `!= 1` Unsure if we show `0 try` but consistently doing this correctly (`0 tries`) makes it more likely that others will be correct in the future (as people might copy).